setup.py

```diff
@@ -68,7 +69,9 @@
     %s = %s:CorePlugin
     [deluge.plugin.gtkui]
```

Comment:
Deluge now uses Gtk 3, so older version class is redundant.

Reply:
I followed the plugin upgrade guide from the developer of Deluge, which suggests leaving the non-3 GTK in. I can definitely remove these for this PR.

Guide: https://deluge.readthedocs.io/en/latest/devguide/how-to/update-1.3-plugin.html

Reply:
>  I can definitely remove these for this PR.
Yes, since you explictly say we're not backward compatible.